scripts/bump_version.py

```diff
@@ -0,0 +1,38 @@
+# This script bumps up the version in setup.py for new releases.
+# Usage: python bump_version.py {new_version} (--read_me <path-to-README.md> --pom_xml <path-to-pom-xml>)
+
+import argparse
+import re
+
+def bump_version(new_version, file_path):
+    with open(file_path, "r") as file:
+        read_me = file.read()
```

Comment:
nit: It's only 'README' for one invocation. Generalize variable name.
```suggestion
        content = file.read()
```

Reply:
Yes thank you

Reply:
Updated.